### PR TITLE
DSD-2602 created script files for bqat 

### PR DESCRIPTION
**Pull request by bhumi46**

### CODE REVIEW

File: deployment/v3/mosip/bqatsdk/install.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Installs BqatSDK
+## Usage: ./install.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+NS=bqatsdk
+CHART_VERSION=12.0.2
```
Comment (ckm007) [inline deployment/v3/mosip/bqatsdk/install.sh:10]: version update required

File: deployment/v3/mosip/bqatsdk/values.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+bqatsdk:
+  ## This is url of bqatsdk lib accessed from within cluster.  BqatSDK may be hosted outside the cluster
+  ## in which case change the URL accordingly.
+  zippedLibUrl: http://artifactory.artifactory/artifactory/libs-release-local/biosdk/bqatsdk-lib.jar
+  bioapiImpl: io.bqat.sdk.services.impl.BqatQualitySdkService
```
Comment (syedsalman3753) [inline deployment/v3/mosip/bqatsdk/values.yaml:5]: Please remove values.yaml as the same values has been added to helm charts values.yaml

File: deployment/v3/mosip/bqatsdk/install.sh [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  echo Istio label
+  kubectl label ns $NS istio-injection=enabled --overwrite
+  helm repo update
+
+  echo Copy configmaps
+  sed -i 's/\r$//' copy_cm.sh
+  ./copy_cm.sh
+
+  echo Installing Bqatsdk server
+  helm -n $NS install bqat-sdk mosip/bqat-sdk -f values.yaml --version $CHART_VERSION
```
Comment (syedsalman3753) [inline deployment/v3/mosip/bqatsdk/install.sh:25]: please remove "f values.yaml" as it is not required here

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: deployment/v3/mosip/bqatsdk/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+## Introduction
+Details about this service are given [here](https://github.com/mosip/mosip-ref-impl/tree/develop/bqat-sdk)
+
+The service expects **bqatsdk library** typically provided by a vendor.  Host the library at a location that can be downloaded from within cluster with a URL. The default installation biosdk service contains mock SDK for testing the interfaces.
+
+## Prerequisites
+To run this BqatSDK service, you will need to first run the [artifactory](../../mosip/artifactory/README.md) which contains the mock SDK library.
+
+## Install
+* Update `values.yaml` with your bqatsdk zipped library url and classpath.
```
Comment (syedsalman3753) [inline deployment/v3/mosip/bqatsdk/README.md:12]: remove this point

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]
can you please tell me y values.yaml has been added to this. As in helm chart we have provided the urls.

**Comment by syedsalman3753** [comment]
@bhumi46 please remove last 3 commits

**Review by syedsalman3753 — APPROVED** [review]

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]
Added the service in all/install-all.sh.

make sure to be in sequence you can check with @ckm007

**Pull request merged**